### PR TITLE
NOD | Increase code coverage

### PR DESCRIPTION
**Pull request by Mottie**
## Summary

- _(Summarize the changes that have been made to the platform)_
  > Add coverage to NOD unit tests: config, migrations, containers, components & reducers
  > We're not fully to 100% in all fields, but slowly working our way up
- _(If bug, how to reproduce)_
- _(What is the solution, why is this the solution)_
- _(Which team do you work for, does your team own the maintenance of this component?)_
  > Benefits Decision Reviews
- _(If using a flipper, what is the end date of the flipper being required/success criteria being targeted)_

## Related issue(s)

[#71500](https://github.com/department-of-veterans-affairs/va.gov-team/issues/71500)

## Testing done

- _Describe what the old behavior was prior to the change_
- _Describe the steps required to verify your changes are working as expected_
- _Describe the tests completed and the results_
- _Exclusively stating 'Specs and automated tests passing' is NOT acceptable as appropriate testing
- _Optionally, provide a link to your [test plan](https://depo-platform-documentation.scrollhelp.site/developer-docs/create-a-test-plan-in-testrail) and [test execution records](https://depo-platform-documentation.scrollhelp.site/developer-docs/execute-tests-in-testrail)_

## Screenshots

|         |   |
| ------- | ----- |
| Before  | before utils code coverage 92.15% statements, 79.39% branches, 84.81% functions and 92.35% lines |
| After | after utils code coverage: 96.07% for statements, 87.3% branches, 89.87% functions and 96.01% lines |

## What areas of the site does it impact?

NOD

### Quality Assurance & Testing

- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [ ] The vets-website header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

File: src/applications/appeals/10182/tests/config/submit-transformer.unit.spec.js [hunk 1]
```diff
@@ -31,7 +58,7 @@ describe('transform', () => {
   });
 
   it('should transform maximal-test.json with part3 data correctly', () => {
-    const data = { data: { [SHOW_PART3]: true, ...maximalData.data } };
+    const data = { data: { ...maximalData.data, [SHOW_PART3]: true } };
```
Comment (rocio-desantiago) [inline src/applications/appeals/10182/tests/config/submit-transformer.unit.spec.js:61]: Did the order through this one off?

Comment (Mottie) [reply src/applications/appeals/10182/tests/config/submit-transformer.unit.spec.js:61]: yes, any data in the first entry is replaced by data from the second, third, etc...

File: src/applications/appeals/10182/tests/config/submitForm.unit.spec.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+import submitForm from '../../config/submitForm';
+
+import { SHOW_PART3 } from '../../constants';
+
+describe('submitForm', () => {
+  let xhr;
+  let requests;
+
+  beforeEach(() => {
+    xhr = sinon.useFakeXMLHttpRequest();
```
Comment (rocio-desantiago) [inline src/applications/appeals/10182/tests/config/submitForm.unit.spec.js:16]: What's xhr?

Comment (Mottie) [reply src/applications/appeals/10182/tests/config/submitForm.unit.spec.js:16]: Abbreviation for XMLHttpRequest

File: src/applications/appeals/10182/tests/migrations/01-rename-contested-issues.unit.spec.js [hunk 3]
```diff
@@ -1,7 +1,28 @@
 import { expect } from 'chai';
-import renameContestedIssues from '../../migrations/01-rename-contested-issues.js';
+import renameContestedIssues from '../../migrations/01-rename-contested-issues';
 
 describe('NOD rename contestedIsuses migration', () => {
+  it('should return contestedIssues empty array', () => {
```
Comment (rocio-desantiago) [inline src/applications/appeals/10182/tests/migrations/01-rename-contested-issues.unit.spec.js:5]: was this renamed `contested` from `contestable`?

**Review by rocio-desantiago — APPROVED** [review]
Just a few questions. One I'm sure I answered myself.

File: src/applications/appeals/10182/tests/migrations/01-rename-contested-issues.unit.spec.js (hunk 3 above)
Comment (Mottie) [reply src/applications/appeals/10182/tests/migrations/01-rename-contested-issues.unit.spec.js:5]: Yes, it was a migration put in place so we could move the area of disagreement code into the shared folder. Eventually, we'll move the ContestableIssuesWidget from this app into the shared folder.

**Pull request merged**